### PR TITLE
some totally optional suggestions for UI tweaks.

### PR DESCRIPTION
**Pull request by vortexing**
Feel free to hate them, but I took at stab at getting things consistent/pretty.

### CODE REVIEW

File: app/ui_components.R [hunk 1]
```diff
@@ -7,7 +7,7 @@ help_html <- helpText(
         <br>
         <p>
             <b>Discussion/Questions</b><br>
-            <a href="https://fhdata.slack.com/archives/CJFP1NYSZ"><span class="badge bg-secondary"><i class="fa-brands fa-slack"></i> Slack</span></a> #question-and-answer channel
+            <a href="https://fhdata.slack.com/archives/CJFP1NYSZ"><span class="badge bg-secondary"><i class="fa-brands fa-slack"></i> Slack</span></a> #workflow-managers channel
```
Comment (sckott) [inline app/ui_components.R:10]: In the PROOF call we changed the channel to `#question-and-answer`. You want to change it back?

File: app/validators.R [hunk 2]
```diff
@@ -18,6 +18,6 @@ validatorOwnCromwell <- function(x) {
   )
   # URL must be a valid URL via rcromwell::cromwell_version
   iv$add_rule("ownCromwellURL",
-    ~ if (!check_url(.)) "Not a valid Cromwell URL")
+    ~ if (!check_url(.)) "Not a valid URL")
```
Comment (sckott) [inline app/validators.R:21]: But the `check_url` function isn't validating URLs in general, just that it's a valid cromwell server. So a valid url is a little misleading, but if  you're just going for simpler then I guess thats fine

File: app/tab-servers.R [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
+      br(),
+      h5("Current PROOF server information (if live)"),
       uiOutput("proofStatusWorkflowLogDir"),
       uiOutput("proofStatusScratchDir"),
       uiOutput("proofStatusServerTime"),
       uiOutput("proofStatusSlurmJobAccount"),
       br(),
-      h4("Troubleshooting"),
+      br(),
+      h4("Stop your PROOF Server"),
```
Comment (sckott) [inline app/tab-servers.R:30]: Do we want to say "PROOF server" when actually that's more accurately the server that serves the PROOF API? Or is is shortand for "PROOF based Cromwell server" and if so, is it okay to use that shorthand?

File: app/ui_components.R (hunk 1 above)
Comment (sitapriyamoorthi) [reply app/ui_components.R:10]: No strong feelings about this. :)

File: app/validators.R (hunk 2 above)
Comment (sitapriyamoorthi) [reply app/validators.R:21]: Either way is fine! But this would be an update i dont think for the screenshots though...

File: app/tab-servers.R (hunk 3 above)
Comment (sitapriyamoorthi) [reply app/tab-servers.R:30]: PROOF based Cromwell Server is long..... aint it?

File: app/modals.R [hunk 4]
```diff
@@ -61,15 +61,15 @@ cromwellStartModal <- function(failed = FALSE, error = "An error occurred") {
   )
 }
 
-verifyCromwellDeleteModal <- function(failed = FALSE, error = "Woops, an error! Contact DaSL") {
+verifyCromwellDeleteModal <- function(failed = FALSE, error = "Woops, an error! Contact DaSL at wilds@fredhutch.org.") {
   modalDialog(
-    title = "Delete your PROOF Cromwell server",
-    "Permanently delete your PROOF Cromwell server. Although you can't undo this action, you can start up another one anytime!",
+    title = "Stop your PROOF server",
+    "Permanently stop your PROOF server. Although you can't undo this action, you can start up another one anytime!",
```
Comment (sckott) [inline app/modals.R:67]: We should remove the word "Permanently" - makes it sound like they can't get another one

File: app/modals.R [hunk 5]
```diff
@@ -61,15 +61,15 @@ cromwellStartModal <- function(failed = FALSE, error = "An error occurred") {
...
     br(),
     br(),
     textInput(
       inputId = "stopCromwell",
-      label = div(HTML("To confirm deletion, type <em>delete me</em> into the field."))
+      label = div(HTML("To stop your server, confirm deletion by typing the text '<em>delete me</em>' into the field."))
```
Comment (sckott) [inline app/modals.R:72]: I'd remove the quotes, that's what the italics are for

File: app/tab-servers.R
Comment (sckott) [inline app/tab-servers.R:1]: It seems a little busy with these changes - but i'll open an issue to look at it again later

File: app/tab-submission.R
Comment (sckott) [inline app/tab-submission.R:1]: I don't love the two buttons being the same color - i think it's visually a little easier to pick out the action button of greatest interest if it's a different color - but we don't have time to wax intellectual on colors, so good for now

File: app/tab-tracking.R [hunk 6]
```diff
@@ -2,21 +2,19 @@ source("ui_components.R")
 
 tab_tracking <- tabItem(
   tabName = "tracking",
-  fluidRow(h2("Cromwell Workflow Tracking"), align = "center"),
   fluidRow(
```
Comment (sckott) [inline app/tab-tracking.R:5]: All of these 3 inputs being in the same row means that it's a little weird clicking up and down arrows on the far right of the screen with the number being way over to the left of the screen. I'd revert this change

File: app/ui_components.R (hunk 1 above)
Comment (vortexing) [reply app/ui_components.R:10]: There is already a specific channel for workflow managers in FH Data Slack and historically folks using WDL and Cromwell have discussed challenges there.  Either works but it would be easier for us to know we need to respond if it's in the workflow manager channel not in question and answer.  but the group wants it in question-and-answer and are willing to CONSISTENTLY respond to questions there promptly, I don't really care.

File: app/tab-tracking.R (hunk 6 above)
Comment (vortexing) [reply app/tab-tracking.R:5]: Ah, on my they were stacked very nicely.  Happy to revert b/c that sounds dumb looking.

File: app/tab-submission.R
Comment (vortexing) [reply app/tab-submission.R:1]: yeah the prior color collection was pretty inconsistent and wild and sometimes was pretty dark so it was hard to see the text.  I opted for consistent but uninformative colors for the sake of at least being able ot consistently read the text of the button.  UXR will likely fix this first. ;).

File: app/modals.R (hunk 4 above)
Comment (vortexing) [reply app/modals.R:67]: Yeah, you're right, I was trying to soft it by removing delete, but now it does look a little crazy to say "permanently". haha

File: app/tab-servers.R
Comment (vortexing) [reply app/tab-servers.R:1]: Is this a comment on the entire servers page changes?  I would tend to agree but the flow of information is more timely in my mind this way.  having the "RUNNING" show up higher in the page makes people know whether they need to click the button to start a new server or not.

File: app/validators.R (hunk 2 above)
Comment (vortexing) [reply app/validators.R:21]: Perhaps HERE is where we write "Not a valid server URL" adn leave it silent as to whether it's a PROOF based server or a DIY Cromwell server?

File: app/ui_components.R (hunk 1 above)
Comment (sckott) [reply app/ui_components.R:10]: okay i think workfow manager is good

File: app/validators.R (hunk 2 above)
Comment (sckott) [reply app/validators.R:21]: Sure sounds good

File: app/tab-servers.R (hunk 3 above)
Comment (sckott) [reply app/tab-servers.R:30]: Okay proof it is

File: app/modals.R (hunk 4 above)
Comment (sckott) [reply app/modals.R:67]: okay, cool

File: app/tab-servers.R
Comment (sckott) [reply app/tab-servers.R:1]: > Is this a comment on the entire servers page changes?

Yes, the entire page. As in, my eyes are a little confused as to what's what. I think it just needs a little finessing with spacing, etc. but I don't feel strongly enough to change anything now

File: app/tab-tracking.R (hunk 6 above)
Comment (sckott) [reply app/tab-tracking.R:5]: on second thought we can set the width of the text box, so that might be better

File: app/tab-troubleshoot.R [hunk 7]
```diff
@@ -28,22 +27,21 @@ tab_troublehsoot <- tabItem(
...
     box(
-      width = 12, solidHeader = FALSE, status = "info",
+      width = 12, solidHeader = FALSE, status = "danger",
       collapsible = TRUE, collapsed = FALSE,
       title = "Troubleshoot a Workflow",
       p("When a workflow fails but no jobs were started, or there appears to be no clear reason for a workflow to have failed, this tool can provide you the entire set of workflow metadata Cromwell has for your workflow in it's raw and unprocessed (json) form. For complex workflows, this can be rather large (and ugly!)."),
       textInput(
         inputId = "troubleWorkflowID",
-        label = "Cromwell workflow id to get metadata for:",
+        label = "PROOF workflow id to get metadata for:",
```
Comment (sckott) [inline app/tab-troubleshoot.R:36]: Better to just say 

> Workflow id to get metadata for:

consistent with abort text and is generic for those not using proof. i'll change that
